setup.py

```diff
@@ -31,7 +31,9 @@
       author_email="jelmer@jelmer.uk",
       license="GNU GPLv3 or later",
       url="https://www.jelmer.uk/projects/xandikos",
-      install_requires=['icalendar', 'dulwich', 'defusedxml'],
+      install_requires=[
+          open('requirements.txt').readlines()
```

Comment:
I'd prefer to keep things simple; I don't use pip myself, but rely on Debian packages so am unlikely to keep a requirements.txt up to date. There shouldn't be a need for xandikos to have more than a handful of dependencies (if that changes, we can revisit this). 

Can you drop requirements.txt for the moment, and update the reference to it in README.md to 
use 'setup.py develop' ?